test/denormalized_has_many_test.rb

```diff
@@ -82,6 +82,11 @@ def test_cache_without_guessable_inverse_name_does_not_raise_when_inverse_name_s
     end
   end
 
+  def test_cache_uses_inverse_of_on_association
+    Item.has_many :invertable_association, :inverse_of => :owner, :class_name => 'PolymorphicRecord', :as => "owner", :inverse_of => :owner
+    Item.cache_has_many :invertable_association, :embed => true
```

Comment:
Sure, see `test_cache_without_guessable_inverse_name_raises` above which tests that case.  rails doesn't require the inverse association.